appengine-java8/endpoints-v2-skeleton/pom.xml

```diff
@@ -0,0 +1,124 @@
+<!--
+ Copyright 2017 Google Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>war</packaging>
+    <version>1.0-SNAPSHOT</version>
+
+    <groupId>com.example.skeleton</groupId>
+    <artifactId>endpoints-j8-skeleton</artifactId>
+
+    <parent>
```

Comment:
If we are providing this as a skeleton app, maybe we should take out the parent reference here.

Reply:
I forgot to add this sample to the parent pom.xml.. The rationale for keeping it in is for dependency updates as IIRC it's dependent on the parent pom.

Reply:
looks like @kurtisvg is working on a related cleanup, so ignore my comment here.